Packages.props

```diff
@@ -22,7 +22,7 @@
 
 		<PackageReference Update="Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider" Version="1.1.1" />
 		<PackageReference Update="Microsoft.SqlServer.Management.SmoMetadataProvider" Version="170.12.0" />
-		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="161.8406.0-preview" />
+		<PackageReference Update="Microsoft.SqlServer.DacFx" Version="162.0.14" />
```

Comment:
the version I put in nuget is [162.0.15-preview](https://www.nuget.org/packages/Microsoft.SqlServer.DacFx/162.0.15-preview) not sure where you are getting 162.0.14 from

Reply:
So that one was from the testing I was doing. I have updated the version to 162.0.15-preview now.